### DISCUSSION TITLE
How can I create hedging strategies on Pybroker?

### DISCUSSION THREAD

**Discussion by Siqueira-J-S** [edtechre/pybroker#142] [Q&A]
Hello, how are you?
I am a new user of this library, which I enjoy, but I have a question.

I am working on a project in which I intend to hedge using futures for a specific asset, and I would like to understand how I can create hedging strategies using Pybroker.

The idea is to develop an ML model to predict returns, create the hedging strategy, and then backtest this strategy.

**Comment by edtechre** [comment]
Hi @Jonas-Santos-Siqueira,

Can you be more specific about what you're looking for? You can short by using [ctx.sell_shares](https://www.pybroker.com/en/latest/reference/pybroker.context.html#pybroker.context.ExecContext.sell_shares).

**Reply by Siqueira-J-S** [reply]
Well, I want to trade put and call options depending on the fluctuation of an asset's returns.
I may not understand how to create this strategy using the library. If you understand my question better, could you help me?

**Reply by edtechre** [reply]
Hi @Jonas-Santos-Siqueira,

PyBroker does not support options at this time.

**Reply by Siqueira-J-S** [reply]
But with this function that you suggested to me, would I be able to operate short on a share?

**Reply by edtechre** [reply]
@Jonas-Santos-Siqueira Yes, you can use it to short shares.
